src/sqlfluff/rules/L057.py

```diff
@@ -57,16 +58,29 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
         self.unquoted_identifiers_policy: str
         self.allow_space_in_identifier: bool
         self.additional_allowed_characters: str
+        self.ignore_words: str
 
         # Exit early if not a single identifier.
         if context.segment.name not in ("naked_identifier", "quoted_identifier"):
             return None
 
+        # Get the ignore_words_list configuration.
+        try:
+            ignore_words_list = self.ignore_words_list
+        except AttributeError:
+            # First-time only, read the settings from configuration. This is
+            # very slow.
+            ignore_words_list = self._init_ignore_words_list()
+
         # Assume unquoted (we'll update if quoted)
         policy = self.unquoted_identifiers_policy
 
         identifier = context.segment.raw
 
+        # Skip if in ignore list
+        if ignore_words_list and identifier in ignore_words_list:
```

Comment:
Is this intended to be case insensitive? I notice we are lowercasing the strings in the list, but apparently not `identifier`. Seems like we should do both or neither.

Reply:
Good shout. Will add.

Reply:
Added